dozer-api/src/grpc/types_helper.rs

```diff
@@ -86,17 +85,22 @@ fn field_to_prost_value(f: Field) -> Value {
         Field::UInt(n) => Value {
             value: Some(value::Value::UintValue(n)),
         },
+        Field::U128(n) => Value {
+            value: Some(value::Value::StringValue(n.to_string())),
```

Comment:
I'm not sure about this. Should it be `Value::U128Value`?

Reply:
yes yes changed